diffeqpy/de.py

```diff
@@ -0,0 +1,15 @@
+from . import setup
+_jul = setup()
+
+# Suppress many UndefVarError printed out.
+# See: https://github.com/JuliaPy/pyjulia/issues/159
+#      https://github.com/JuliaDiffEq/diffeqpy/pull/24
+try:
```

Comment:
so what exactly is this doing?

Reply:
The problematic part in pyjulia is

```py
self.api.jl_call2(self.api.show, stderr, exoc)
```

in `Julia.check_exception` (https://github.com/JuliaPy/pyjulia/blob/2eb704c33d6f639b41300023e16f910b5945600e/julia/core.py#L416-L432) which is called unconditionally before `raise JuliaError`.  The `JuliaError` is suppressed by try-except in pyjulia module loader but call to `Base.show` above is not.

What I'm doing here is to monkey-patch `self.api` so that `self.api.show` points to Julia's `identity` function instead of `show`.

Wait, `identity` does not take two arguments. Why does it work?! I should probably use other function. Fixing it...

Reply:
I guess fix this up to something that's supposed to work (?) and we're good?